src/dev.sh

```diff
@@ -119,7 +119,7 @@ function cmd_package ()
         echo "You must build first.  Expecting to find ${LAYOUT_DIR}/bin"
     fi
 
-    agent_ver=$("${LAYOUT_DIR}/bin/Agent.Listener" --version | tail -n 1) || failed "version"
+    agent_ver=$(cat "${SCRIPT_DIR}/agentversion" | tail -n 1) || failed "version"
```

Comment:
what are these changes?

Reply:
slightly unrelated change. I was trying to build a linux build from my Mac and needed these changes to do it.